KUBERNETES.md

```diff
@@ -0,0 +1,241 @@
+
+##### Start minikube
+
+> If you have an existing Minikube, you can delete your old one start fresh via `minikube delete`
+
+##### Enable Ingress Controller
+
+```bash
+minikube addons enable ingress
+```
+
+```bash
+minikube start --memory 6000
+```
+
+##### Setup Docker engine context to point to Minikube
+
+```bash
+eval $(minikube docker-env)
+```
+
+#### 2a. Development Workflow
+
+> Note that this is an alternative to the Operations workflow documented below.
+
+`sbt-reactive-app` defines a task, `deploy minikube`, that can be used to deploy all aggregated subprojects to
+your running Minikube. It also installs the [Reactive Sandbox](https://github.com/lightbend/reactive-sandbox/) if
+your project needs it, e.g. for Lagom applications that use Cassandra or Kafka.
+
+##### Build and Deploy Project
+
+`sbt> deploy minikube`
+
+Once completed, Chirper and its dependencies should be installed in your cluster. Continue with step 3, Verify Deployment.
+
+#### 2b. Operations Workflow
+
+> Note that this is an alternative to the Development workflow documented above.
+
+##### Install Reactive Sandbox
+
+The `reactive-sandbox` includes development-grade (i.e. it will lose your data) installations of Cassandra, Elasticsearch, Kafka, and ZooKeeper. It's packaged as a Helm chart for easy installation into your Kubernetes cluster.
+
+> Note that if you have an external Cassanda cluster, you can skip this step. You'll need to change the `cassandra_svc` variable (defined below) if this is the case.
+
+```bash
+helm init
+helm repo add lightbend-helm-charts https://lightbend.github.io/helm-charts
+helm update
+```
+
+Verify that Helm is available (this takes a minute or two):
+
+```bash
+kubectl --namespace kube-system get deploy/tiller-deploy
+```
+
+```
+NAME            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+tiller-deploy   1         1         1            1           3m
+```
+
+Install the sandbox. Since Chirper only uses Cassandra, we're disabling the other services but you can leave them enabled by omitting the `set` flag if you wish.
+
+```bash
+helm install lightbend-helm-charts/reactive-sandbox --name reactive-sandbox --set elasticsearch.enabled=false,kafka.enabled=false,zookeeper.enabled=false
+```
+
+Verify that it is available (this takes a minute or two):
+
+```bash
+kubectl get deploy/reactive-sandbox
+```
+
+```
+NAME               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+reactive-sandbox   1         1         1            1           1m
+```
+
+##### Build Project
+
+```bash
+sbt clean docker:publishLocal
+```
+
+##### View Images
+
+```bash
+docker images
+```
+
+##### Deploy Projects
+
+Finally, you're ready to deploy the services. Be sure to adjust the secret variables and cassanda service address as necessary.
+
+```bash
+# Be sure to change these secret values
+
+chirp_secret="youmustchangeme"
+friend_secret="youmustchangeme"
+activity_stream_secret="youmustchangeme"
+front_end_secret="youmustchangeme"
```

Comment:
We should make these `changeme` - see the discussion here: https://github.com/lagom/online-auction-scala/pull/62/files/fa14ccdca11a3a3a1ab49590f80463b190d747de#r169196711